bin/composer-scripts/ProjectEvents/PostInstallScript.php

```diff
@@ -628,6 +628,7 @@ public static function renderSuccessMessage(): void {
 		$success = [
 			'Congratulations!',
 			'Project has been set up successfully!',
+			'Admin Username: viget',
```

Comment:
Since we're [hard-coding the admin username](https://github.com/vigetlabs/wordpress-site-starter/blob/main/bin/composer-scripts/ProjectEvents/PostInstallScript.php#L287) when creating the initial user, it would probably be best to set a constant in `PostInstallScript` (`const DEFAULT_ADMIN_USER = 'viget'`) and use that variable when creating the user as well as in the success message.

Even one step further might be add a step to the `getSiteInfo()` method [just before collecting the Admin Email](https://github.com/vigetlabs/wordpress-site-starter/blob/main/bin/composer-scripts/ProjectEvents/PostInstallScript.php#L318) to allow the Username to be set during installation (using the new constant as the default).

Reply:
Thanks for the links in the code! Made it easy to update and allow the username to be set during install!